modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModalAddObjectField.tsx

```diff
@@ -156,7 +159,16 @@ export default function ModalWithProvider({
 			{isVisible && (
 				<ModalAddObjectField
 					apiURL={apiURL}
-					objectFieldTypes={objectFieldTypes}
+					objectDefinitionId={objectDefinitionId}
+					objectFieldTypes={
+						!Liferay.FeatureFlags['LPS-149625']
```

Comment:
@naraandrade, No, we have to have this '!' here the reason for this is because Frontend will get the new FieldType always, so when the feature flag is disabled we have to filter this new type not to show it in the modal